### PR TITLE
KAZOO-5014 decode to field of cdr legs

### PR DESCRIPTION
**Pull request by jamhed**

### CODE REVIEW

File: applications/crossbar/src/modules/cb_cdrs.erl [hunk 1]
```diff
@@ -777,4 +777,9 @@ load_legs(InteractionId, Context) ->
 -spec normalize_leg_view_results(kz_json:object(), kz_json:objects()) ->
                                         kz_json:objects().
 normalize_leg_view_results(JObj, Acc) ->
-    Acc ++ [kz_json:get_value(<<"doc">>, JObj)].
+    Acc ++ [fix_leg_values(kz_json:get_value(<<"doc">>, JObj))].
```
Comment (fenollp) [inline applications/crossbar/src/modules/cb_cdrs.erl]: Would it be possible to flip the arguments of `++` here? just in order to save copying `Acc`.

**Comment by lazedo** [comment]
@jamhed your are trying to fix the effect not the cause

**Comment by jamhed** [comment]
@lazedo, please explain the cause.

**Comment by lazedo** [comment]
fixed elsewhere

**Pull request closed without merging**